packages/accounts-password/password_client.js

```diff
@@ -7,6 +7,31 @@ const reportError = (error, callback) => {
    }
 };
 
+
+const internalLoginWithPassword = ({ selector, password, token, callback }) => {
```

Comment:
Maybe this should be exposed in the accounts object as `_internalLoginWithPassword` so `Meteor.loginWithPasswordAnd2faToken` could be present in the correct package instead of here.

Do you agree with that @denihs ?

Reply:
My problem with that is when we decide to integrate 2FA with another login method, like passwordless, we'll need to change both packages (2fa and passwordless) to keep the same logic, whereas if we keep it as it is right now, we'll need to change just the passwordless.

This would be my only argument. But we can do as you suggested, you prefer.

Reply:
I don't see how.

If we add new packages they can also use _internalLoginWithPassword as 2fa is going to use now.

I believe we should change that.